From 6c8ff356d7cbc8235bade794a248ee8fb545739a Mon Sep 17 00:00:00 2001
From: Sean Paul <seanpaul@chromium.org>
Date: Mon, 12 May 2014 16:25:46 -0400
Subject: [PATCH 18/18] HACK: drm/i915: Remove WARN for mode_config.mutex in
 ironlake_panel_vdd_off_sync

BUG=chromium:336809
TEST=Tested on snow & peppy

Change-Id: I155bf463e04929c7e1bfec7842da386f35697450
Signed-off-by: Sean Paul <seanpaul@chromium.org>
---
 drivers/gpu/drm/i915/intel_dp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_dp.c b/drivers/gpu/drm/i915/intel_dp.c
index 6dbbf6c..2ead490 100644
--- a/drivers/gpu/drm/i915/intel_dp.c
+++ b/drivers/gpu/drm/i915/intel_dp.c
@@ -986,8 +986,6 @@ static void ironlake_panel_vdd_off_sync(struct intel_dp *intel_dp)
 	struct drm_i915_private *dev_priv = dev->dev_private;
 	u32 pp;
 
-	WARN_ON(!mutex_is_locked(&dev->mode_config.mutex));
-
 	if (!intel_dp->want_panel_vdd && ironlake_edp_have_panel_vdd(intel_dp)) {
 		pp = ironlake_get_pp_control(dev_priv);
 		pp &= ~EDP_FORCE_VDD;
-- 
1.8.3.2

